src/sinks/redis.rs

```diff
@@ -124,14 +124,17 @@ pub struct RedisSinkConfig {
     #[serde(alias = "list")]
     list_option: Option<ListOption>,
 
-    /// The Redis URL to connect to.
+    /// The URL of the Redis endpoint to connect to.
     ///
     /// The URL _must_ take the form of `protocol://server:port/db` where the protocol can either be
     /// `redis` or `rediss` for connections secured via TLS.
-    url: String,
+    #[configurable(metadata(docs::examples = "redis://127.0.0.1:6379/0"))]
+    #[serde(alias = "url")]
+    endpoint: String,
```

Comment:
❤️ love this change, but I think it also needs a note in the upcoming upgrade guide?

Reply:
Done.